packages/ember-metal/lib/property_get.js

```diff
@@ -60,35 +56,18 @@ export function get(obj, keyName) {
     return obj;
   }
 
-  var meta = peekMeta(obj);
-  var value = obj[keyName];
-  var desc = (value !== null && typeof value === 'object' && value.isDescriptor) ? value : undefined;
-  var ret;
-
-  if (desc === undefined && isPath(keyName)) {
+  if (isPath(keyName)) {
```

Comment:
needs testing to be sure checking descs isn't cheaper then this. but keeping it simpler at first is likely better, but we should investigate further.